stdlib/public/core/UnicodeScalarProperties.swift

```diff
@@ -692,15 +692,16 @@ extension Unicode.Scalar.Properties {
   /// all current case mappings. In the event more space is needed, it will be
   /// allocated on the heap.
```

Comment:
This doc comment seems outdated. IIUC, a buffer capable of holding 64 code units is allocated on the stack with this change, whereas as originally designed only 16 code units would have been allocated there, with the rest on the heap if needed.

Reply:
That absolutely makes sense. I've changed a little the second part.